MSSQLDatabaseContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace InventorySystem;
+
+public class MSSQLDatabaseContext : DbContext
+{
+    public DbSet<Product> Product { get; set; }
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        optionsBuilder.UseSqlServer(@"Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=master;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False");
```

Comment:
Note for the future: you should never include a connection string directly in your code.